helm/payment-hub-barebone/templates/elephant-connector-mojaloop-java-ingress.yml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
Why are we modifying elephant connector?

Reply:
extensions/v1beta1 is deprecated.